client/extensions/woocommerce/state/ui/payments/methods/actions.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Internal dependencies
+ */
+import {
+	WOOCOMMERCE_PAYMENT_METHOD_CANCEL,
+	WOOCOMMERCE_PAYMENT_METHOD_CLOSE,
+	WOOCOMMERCE_PAYMENT_METHOD_EDIT_FIELD,
+	WOOCOMMERCE_PAYMENT_METHOD_OPEN,
+} from '../../../action-types';
+
+export const openPaymentMethodForEdit = ( siteId, id ) => {
+	return { type: WOOCOMMERCE_PAYMENT_METHOD_OPEN, siteId, id };
+};
+
+export const closeEditingPaymentMethod = ( siteId ) => {
```

Comment:
I'm not sure how I feel about edit state being tied so closely to UI state like this. I don't think this has to change now, but I do think we should put some more thought to consolidating this process, perhaps after v1 release.